packages/astro/e2e/fixtures/view-transitions/src/pages/one.astro

```diff
@@ -12,6 +12,11 @@ import Layout from '../components/Layout.astro';
 	<a id="click-redirect-two" href="/redirect-two">go to redirect 2</a>
 	<a id="click-redirect-external" href="/redirect-external">go to a redirect external</a>
 	<a id="click-404" href="/undefined-page">go to undefined page</a>
+	<custom-a id="custom-click-two">
+		<template shadowrootmode="open">
```

Comment:
I'm not familiar with custom elements, but is it required to have the ` shadowrootmode` set to "open" for this to work? If that's the case it would be a good idea documenting it

Reply:
In the changeset, you mean?

Reply:
I meant somewhere in the VT docs as a note/warning maybe

Reply:
You're right. I could make a PR to docs after this.